code/modules/mob/living/simple_animal/abnormality/he/schadenfreude.dm

```diff
@@ -0,0 +1,100 @@
+/mob/living/simple_animal/hostile/abnormality/schadenfreude
+	name = "Schadenfreude"
+	desc = "A box with a keyhole. You don't want to know what's inside."
+	icon = 'ModularTegustation/Teguicons/48x48.dmi'
+	icon_state = "schadenfreude"
+	icon_living = "schadenfreude"
+	pixel_x = -8
+	base_pixel_x = -8
+	del_on_death = TRUE
+	maxHealth = 2400		//It's fucking slow as hell
+	health = 2400
+	move_to_delay = 6
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.6, WHITE_DAMAGE = 0.2, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 0.7)
+	melee_damage_lower = 40		//Yeah it's super slow, and you're not gonna get hit by it too often
+	melee_damage_upper = 48
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/abnormalities/scarecrow/attack.ogg'
+	attack_verb_continuous = "bashes"
+	attack_verb_simple = "bash"
+	faction = list("hostile")
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	start_qliphoth = 4
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 0,
+						ABNORMALITY_WORK_INSIGHT = list(30, 40, 40, 50, 50),
+						ABNORMALITY_WORK_ATTACHMENT = list(40, 40, 40, 30, 20),
+						ABNORMALITY_WORK_REPRESSION = list(40, 45, 50, 55, 60)
+						)
+	work_damage_amount = 7
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/gaze,
+		/datum/ego_datum/armor/gaze
+		)
+//	gift_type =  /datum/ego_gifts/gaze
+
+	var/seen	//Are you being looked at right now?
+
+//Sight Check
+/mob/living/simple_animal/hostile/abnormality/schadenfreude/Life()
+	. = ..()
+	//Make sure there actually are two players on the Z level
+	var/living_players
+	for(var/mob/living/carbon/human/H in GLOB.player_list)
+		if(H.z == z && H.stat != DEAD)
+			living_players +=1
+	if(living_players == 1)
+		seen = TRUE
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 1.0, PALE_DAMAGE = 1.5)
+		return
+
+	//Who is watching us
+	var/people_watching
+	for(var/mob/living/carbon/human/L in viewers(world.view + 1, src))
+		if(L.client && CanAttack(L) && L.stat != DEAD)
+			if(!L.is_blind())
+				people_watching+=1
+
+	//Only gets mad if there are two people looking at you. If there are 3 or more the counter decreases.
+	if(people_watching >= 3)
+		datum_reference.qliphoth_change(-1)
```

Comment:
might want a cooldown on this one in particular, because depending on how fast Life() runs, this could result in someone walking by the room and it decreasing rapidly